themes/question/bootstrap_buttons/survey/questions/answer/multiplechoice/config.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Also empty.

Reply:
Remove space at end ? Unsure (don't remind)

Reply:
Can be Microsoft vs Unix line ending.

Reply:
Can not : else have it at all lines. Start of the line ? BOM ?